src/main.ts

```diff
@@ -545,6 +546,28 @@ export async function getToken(input: CspInput): Promise<string> {
   }
 }
 
+export async function checkTokenExpiration(): Promise<string> {
+  const response = await cspClient.post("https://console.cloud.vmware.com/csp/gateway/am/api/auth/api-tokens/details", {
```

Comment:
Could we define a constant to centralize the configuration?

Reply:
do you mean a constant like the default base folder or pipeline in `constants.ts`?

Reply:
Actually good catch. You need to remove the domain name and keep just the URL path, [like here](https://github.com/vmware-labs/vmware-image-builder-action/blob/main/src/main.ts#L527).

Then, a good improvement is what @recena suggests, moving the paths (e.g. `/csp/gateway/am/api/auth/api-tokens/details`) to constants, e.g. (`constants.TOKEN_DETAILS_PATH`)

Reply:
It should be done 🙂